package.json

```diff
@@ -17,10 +17,16 @@
     "lint": "eslint \"src/**/*.ts\"",
     "lint:tsc": "tsc --noEmit -p .",
     "docs": "typedoc --excludePrivate --excludeProtected --name \"applemusic.js documentation\" --out docs src/index.ts",
-    "test": "jest --setupFiles dotenv/config"
+    "test": "jest --setupFiles dotenv/config --verbose"
```

Comment:
Do we need the `verbose` flag. If it was for your own debugging I think we can remove it

Reply:
I'd prefer to leave it there, otherwise the output will be shorten